sonar-unit-tests.xsd

```diff
@@ -0,0 +1,54 @@
+<?xml version="1.0"?>
+<xs:schema xmlns:xs="http://www.w3.org/2001/XMLSchema">
```

Comment:
I just saw you mention it is used for testing, but I don't see any tests added for this